### DISCUSSION TITLE
Docker compose command

### DISCUSSION THREAD

**Discussion by udernaeb** [oreplay/oreplay#9] [Show and tell]
`#/bin/bash

if [ -z "$1" ]
  then
    echo "Usage: build-containers.sh env"
    exit 1
fi

if [ ! -f devops/docker/docker-compose-$1.yml ]
  then
    echo "Invalid env. Available options: local | dev. "
    exit 1
fi

docker-compose -f devops/docker/docker-compose.yml -f devops/docker/docker-compose-$1.yml build
`

This is a example of a sh file to easily execute docker compose commands

**Comment by freefri** [comment]
We have this command in the readme file `docker-compose -f ./docker-compose-dev.yml up -d` which is also adding the job. If you are going to use a script like this, we could add it into the files from the project, so you can access to it easily. @udernaeb

**Reply by udernaeb** [reply]
For me, the easy step is that create a sh file, and add it to the code, in order to launch it without writing the command.

**Reply by freefri** [reply]
Yes, this is what I mean. If you find this file useful for you, you can add it into the files from the project.

**Comment by udernaeb** [comment]
File added to `develop` branch
